### DISCUSSION TITLE
"Submit Fail" error when trying to save on ThinkPad X395

### DISCUSSION THREAD

**Discussion by realEnzeta** [DavidS95/Smokeless_UMAF#45] [Q&A]
[![image](https://github.com/DavidS95/Smokeless_UMAF/assets/73473573/e5bc313b-64e6-4764-9e90-d293e95b1d66)](url) 
This "Submit Fail For Form" error appears when trying to save changes in any category (which after rebooting are not applied) and seems to have only been reported on ThinkPad models (but not all Lenovo laptops).

[This guy](https://www.reddit.com/r/GamingLaptops/comments/yvpae5/to_all_amd_laptop_owners_you_can_turn_on_s3_sleep/j7wgysp/
) claims to have gotten the changes to apply by rebooting once the error was shown but this doesn't work for me.

In my case:
ThinkPad X395 (same board as T495s)
RP-3500U
BIOS 1.28 (2023-01-12)
Secure Boot, Device Guard, Memory Encryption and Data Execution Protection are all disabled

**Comment by Atzphere** [comment]
Totally inappropriate place to ask but are you NZZN/NerbyParties on scratch? Just logged in for the first time in years and I remember we used to interact a ton...
